spec/views/repo_spec.rb

```diff
@@ -23,7 +26,7 @@
   end
 
   it 'displays repository rating' do
-    expect(rendered).to have_selector('span', text: repo.rating)
+    expect(rendered).to have_selector('span', text: rating)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.